Mothercell/Hive.IO/Hive.IO/GHComponents/GHVisualizer.cs

```diff
@@ -313,90 +279,162 @@ private void RenderCapsule(GH_Canvas canvas, Graphics graphics)
+            var totalFloorArea = Owner.Results.TotalFloorArea;
+
+            var resultsTotalHeatingMonthly = Owner.Results.TotalHeatingMonthly ?? new double[months];
+            var strokeThickness = 4.0;
 
-            var resultsTotalHeatingHourly = Owner.Results.TotalHeatingHourly ?? new double[hours];
             var demandHeating = new ColumnSeries
             {
-                ItemsSource = resultsTotalHeatingHourly.Select(demand => new ColumnItem { Value = demand }),
-                Title = " Heating Demand"
+                ItemsSource = resultsTotalHeatingMonthly.Select(demand => new ColumnItem { Value = demand / totalFloorArea }),
+                Title = " Space Heating",
+                FillColor = BackgroundColor,
+                StrokeThickness = strokeThickness,
+                StrokeColor = SpaceHeatingColor
             };
             model.Series.Add(demandHeating);
 
-            var resultsTotalCoolingHourly = Owner.Results.TotalCoolingHourly ?? new double[hours];
+            var resultsTotalCoolingMonthly = Owner.Results.TotalCoolingMonthly ?? new double[months];
             var demandCooling = new ColumnSeries
             {
-                ItemsSource = resultsTotalCoolingHourly.Select(demand => new ColumnItem { Value = demand }),
-                Title = " Cooling Demand"
+                ItemsSource = resultsTotalCoolingMonthly.Select(demand => new ColumnItem { Value = demand / totalFloorArea }),
+                Title = " Space Cooling",
+                FillColor = BackgroundColor,
+                StrokeThickness = strokeThickness,
+                StrokeColor = SpaceCoolingColor
             };
             model.Series.Add(demandCooling);
+
+            var resultsTotalElectricityMonthly = Owner.Results.TotalElectricityMonthly ?? new double[months];
+            var demandElectricity = new ColumnSeries
+            {
+                ItemsSource = resultsTotalElectricityMonthly.Select(
+                    demand => new ColumnItem { Value = demand / totalFloorArea }),
+                Title = " Electricity",
+                FillColor = BackgroundColor,
+                StrokeThickness = strokeThickness,
+                StrokeColor = ElectricityColor
+            };
+            model.Series.Add(demandElectricity);
+
+            var resultsTotalDwhMonthly = Owner.Results.TotalDHWMonthly ?? new double[months];
+            var demandDhw = new ColumnSeries
+            {
+                ItemsSource = resultsTotalDwhMonthly.Select(demand => new ColumnItem { Value = demand / totalFloorArea }),
+                Title = " DWH",
+                FillColor = BackgroundColor,
+                StrokeThickness = strokeThickness,
+                StrokeColor = DhwColor,
+            };
+            model.Series.Add(demandDhw);
+
+            model.Axes.Add(new LinearAxis
+            {
+                Position = AxisPosition.Left,
+                Key = "Demand",
+                Title = "kWh/m^2"
```

Comment:
`@"kWh/m²"`

Reply:
I took this from the design specs?

Reply:
it's just a detail, but I think it looks nicer when it displays m² instead of m^2 on the graph in grasshopper